keymanapp/keyman
# fix(mac): limit short bundle version string to x.y.z format in info.plist

In the info.plist for both the Keyman Engine and Input Method, remove the framing text so that CFBundleShortVersionString just contains the version string, x.y.z where x, y and z are numeric values.

Fixes: #12197
Fixes: #12130

For background, see discussion in PR #12156 which was addressing the side effect rather than the root cause.

# User Testing

## TEST_DOWNLOAD_KEYBOARD_VISIBLE_MONTEREY

1. With a mac running macOS Monterey, select the Configuration... command from the Keyman menu
2. In the Keyboards pane, click the Download Keyboard... button
3. Confirm that the Download Keyboard dialog should appear as shown:
![image](https://github.com/user-attachments/assets/f7d16435-5b42-4d08-8613-5c2cb27365d1)

## TEST_DOWNLOAD_KEYBOARD_VISIBLE_OTHER_OS

1. With a mac running a macOS version other than Monterey, select the Configuration... command from the Keyman menu
2. In the Keyboards pane, click the Download Keyboard... button
3. Confirm that the Download Keyboard dialog appears as shown above.

## TEST_DOWNLOAD_AND_INSTALL_KEYBOARD

1. Download and install a new keyboard after invoking the Configuration... command 
2. Confirm that the downloaded keyboard is installed successfully and can be used
